src/main/java/uk/co/ramp/covid/simulation/population/Pensioner.java

```diff
@@ -1,5 +1,25 @@
 package uk.co.ramp.covid.simulation.population;
 
+import uk.co.ramp.covid.simulation.CovidParameters;
+import uk.co.ramp.covid.simulation.DailyStats;
+
 public class Pensioner extends Person {
```

Comment:
Would it be more logical for pensioners to extend Adult?

Reply:
Probably not since we don't want to inherit professions etc. They are treated pretty differently throughout.